csvcubed/shacl/v1/0/deps/shacl-context.json

```diff
@@ -0,0 +1,44 @@
+{
+    "@context": [
+        "https://raw.githubusercontent.com/w3c/shacl/master/shacl-jsonld-context/shacl.context.ld.json",
+        {
+            "uniqueLang": {
+                "@id": "sh:uniqueLang",
+                "@type": "xsd:boolean"
+            },
+            "description": {
+                "@id": "sh:description",
+                "@type": "xsd:string"
+            },
+            "name": {
+                "@id": "sh:name",
+                "@type": "xsd:string"
+            },
+
+            "skos": "http://www.w3.org/2004/02/skos/core#",
+            "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+            "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+            "xsd": "http://www.w3.org/2001/XMLSchema#",
+            "dct": "http://purl.org/dc/terms/",
+            "owl": "http://www.w3.org/2002/07/owl#",
+            "dcat": "http://www.w3.org/ns/dcat#",
```

Comment:
No, since pmdcat should only show up once we've converted the CSV-W into RDF that's suitable for upload to PMD.

So the CSV-Ws should never refer to PMDCAT or similar.